api/config/v1alpha1/envoyproxy_types.go

```diff
@@ -105,6 +109,85 @@ type ProxyLogging struct {
 	Level map[LogComponent]LogLevel `json:"level,omitempty"`
 }
 
+type ProxyAccessLogging struct {
+	// Format defines the format of access logging.
+	Format ProxyAccessLoggingFormat `json:"format"`
+	// Sink defines the sink of access logging.
+	Sink ProxyAccessLoggingSink `json:"sink"`
```

Comment:
what about make this a slice? 
cc @arkodg

Reply:
the top level is already a slice, so dont see much benefit, unless you feel like the format is so complex, thats users dont want to replicate it